net/lagrange/Portfile

```diff
@@ -35,6 +37,9 @@ depends_lib-append  port:libsdl2 \
 compiler.c_standard 2011
 compiler.blacklist-append {clang < 800}
 
+configure.args-append "-DENABLE_HARFBUZZ_MINIMAL:BOOL=OFF"
+configure.args-append "-DENABLE_FRIBIDI_BUILD:BOOL=OFF"
+
```

Comment:
you add the dependencies for harbuzz and fribidi, but you don't want to enable them? Or do I misread the intention of these flags?

Reply:
No, these flags say to use system libs instead of building them.

Reply:
fair enough - thanks!